src/test/java/igrad/logic/commands/DeleteCommandTest.java

```diff
@@ -0,0 +1,108 @@
+package igrad.logic.commands;
+
+import static igrad.logic.commands.CommandTestUtil.assertCommandFailure;
+import static igrad.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static igrad.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
```

Comment:
i think  you removed the last 3 lines of static imports as well, please use static imports, they're much cleaner;

```
import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
```